### DISCUSSION TITLE
Gerader Strich mit Maßpfeil

### DISCUSSION THREAD

**Discussion by 171252** [tannerhelland/PhotoDemon#614] [Q&A] [answered]
Hallo,

wie kann ich einen geraden Strich mit einem Maßpfeil (oder zwei Maßpfeile wahlweise) erzeugen?

**Comment by hi5** [comment, marked as answer]
At the moment it is not yet possible to draw arrows, but it has been suggested and perhaps we'll see it in PD at some point in the future.
Reference:
https://github.com/tannerhelland/PhotoDemon/issues/271
https://github.com/tannerhelland/PhotoDemon/issues/597

**Reply by 171252** [reply]
Ok, besten Dank !

**Reply by tannerhelland** [reply]
@hi5 is correct.  Thank you for this suggestion, @171252 .  I hope it will not be too long before I can get these tools added for you!
